### PR TITLE
MNT Update @methods on class docblocks

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/.github/issues/163

### CODE REVIEW

File: src/Actions/UnpublishItemWorkflowAction.php [hunk 1]
```diff
@@ -10,8 +10,8 @@
 use Symbiote\AdvancedWorkflow\DataObjects\WorkflowInstance;
 use Symbiote\AdvancedWorkflow\Extensions\WorkflowEmbargoExpiryExtension;
 use Symbiote\AdvancedWorkflow\Jobs\WorkflowPublishTargetJob;
-use Symbiote\QueuedJob\Services\AbstractQueuedJob;
-use Symbiote\QueuedJob\Services\QueuedJobService;
+use Symbiote\QueuedJobs\Services\AbstractQueuedJob;
+use Symbiote\QueuedJobs\Services\QueuedJobService;
```
Comment (ChloeSartorelli) [inline src/Actions/UnpublishItemWorkflowAction.php:14]: This was already done in a separate PR - but should be okay here too

**Review by GuySartorelli — APPROVED** [review]
LGTM
